Content.Client/Chat/ChatManager.cs

```diff
@@ -16,6 +18,16 @@ internal sealed class ChatManager : IChatManager
         private const char OOCAlias = '[';
         private const char MeAlias = '@';
 
+        public List<MsgChatMessage> filteredHistory = new List<MsgChatMessage>();
```

Comment:
Still shouldn't be storing `MsgChatMessage` directly.

Reply:
Created a new class to store messages locally that just maintains Message(text), Channel, and MessageWrap. Submits this to its own Repopulate method that draws/clears the chat contents independent of `_onChatMessage`